tests/installation/installation_overview.pm

```diff
@@ -45,7 +45,10 @@ sub run() {
 
     $self->check_and_record_dependency_problems;
 
-    if (check_var('ARCH', 's390x') && !get_var('UPGRADE')) {    # s390x always needs SSH
+    my $need_ssh = check_var('ARCH', 's390x');    # s390x always needs SSH
+    $need_ssh = 1 if check_var('BACKEND', 'ipmi');    # we better be able to login
+
+    if (!get_var('UPGRADE') && $need_ssh) {
```

Comment:
if ((check_var('ARCH', 's390x') && !get_var('UPGRADE')) ||  check_var('BACKEND', 'ipmi') )
this form looks more clear for me